data/random-teams.js

```diff
@@ -1663,6 +1674,13 @@ class RandomTeams extends Dex.ModdedDex {
 			} else if (tier === 'PU') {
 				puCount++;
 			}
+		}
+		while (pokemon.length < 6) {
+			let set = this[this.format.gameType === 'singles' ? 'randomSet' : 'randomDoublesSet'](team.pop(), pokemon.length, teamDetails);
+			pokemon.push(set);
+
+			// Illusion shouldn't be the last Pokemon of the team
+			if (set.ability === 'Illusion' && pokemon.length > 4) pokemon[0] = pokemon.splice(pokemon.length - 1, 1, pokemon[0])[0];
```

Comment:
This is not a correct thing to do. Pokemon get Focus Sash based on being in slot 0.